altair_saver/savers/_selenium.py

```diff
@@ -160,6 +160,7 @@ def __init__(
         spec: JSONDict,
         mode: Optional[str] = None,
         embed_options: Optional[JSONDict] = None,
+        vega_cli_options: Optional[List] = None,
```

Comment:
Unused here

Reply:
See comment above re: `HTMLSaver`. Either this needs to just be passed via the `kwargs` or if passed explicitly in the `__init__` for all the subclasses. 

Happy to refactor.

Reply:
This is refactored in the 3rd commit. Can be merged w/o that or not.

Reply:
This has been removed.